charter/index.html

```diff
@@ -92,11 +96,12 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
         <p>Any text rendered like <span class="todo">this</span> refers to content that must be updated when the final charter is published at the latest (e.g., adjusting hyperlinks).</p>
       </div>
 
-      <p class="mission">The <strong>mission</strong> of the <a href="">Solid Working Group <i class="todo">(LINK TBD)</i></a> is to standardize a protocol for user-centric data and its use of associated data interoperability and authentication schemes.
-      The resulting open standards intends to ensure interoperability and foster development in a decade-old ecosystem of application and server developers, aiming to return control of data back to users.</p>
+      <p class="mission new">The <strong>mission</strong> of the <a href="">PUMPKIN Working Group <i class="todo">(LINK TBD)</i></a> is to 
+      enable the development of web applications where data storage, authentication, and access control, are decoupled from the application provider.
```

Comment:
```suggestion
      enable the development of web applications where data storage, entity authentication, access control, and application provider are all loosely coupled, as opposed to the web of today where these are typically all tightly coupled and changing one requires changing all, sometimes at the price of all past data.
```

Reply:
> FWIW, I think it's fine if the resulting protocol doesn't _guarantee_ decoupling, and the mission agrees with that by only promising that the protocol will _enable_ decoupling.

Is that really the mission?  Because there are existing examples of storage and identity loosely coupled on the web, such as using authentication headers.

What is left in the statement, is a some ways to add access control using Linked Data and Turtle which is loosely-coupled ie ACL and ACP.  Though this could be done with a standard policy too, e.g. everyone can read, owner can write.

This is helpful:

https://timbl.solidcommunity.net/2023/03%20EWADA/TAGExplainerforSolid.html

It says something about data portability (e.g. using Linked Data).  But if you consider the web and solid, the protocols can be packaged together, or individually.  This doesnt guarantee that all apps are client side (ultimately they wont be able to be at scale), for example nextcloud could perfectly use solid but not have client side apps.

A good clear mission would be quite helpful in the document.  For example explaining things that Solid can enable that are not already enabled on the web, such as leading towards a web operating system.  Or creating something on top of LDP to allow it to do more.  Or creating a first class HTTP Identity system, that is compliant with existing best practices.  I acknowledge that there will be multiple different missions of solid, but good to get a common understanding that is accurate.